docs/admin/pool_operations.md

```diff
@@ -44,12 +44,12 @@ $ dmg pool create --help
       -P, --properties= Pool properties to be set
       -a, --acl-file=   Access Control List file path for DAOS pool
       -z, --size=       Total size of DAOS pool (auto)
-      -t, --tier-ratio= Distribution of pool storage allocation over storage tiers (auto) (default: 6)
+      -t, --tier-ratio= Percentage of SCM,NVMe storage tiers for pool storage (auto) (default: 6,94)
```

Comment:
My only hesitation with this line is that the wording was deliberately changed when we added support for N storage tiers vs. a fixed set of SCM/NVMe tiers. 

That having been said, in practice at the moment we still only support SCM and NVMe, so perhaps a compromise would be to change the wording to say `Distribution of total pool space over storage tiers (auto) (default: 6% SCM, 94% NVMe)`?

Reply:
On master, this is the output:
`-t, --tier-ratio=   Percentage of storage tiers for pool storage (auto) (default: 6,94)`

Reply:
ok, fixed